libs/counterfactuals/src/util/getOriginalData.ts

```diff
@@ -1,26 +1,43 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { IDataset, JointDataset } from "@responsible-ai/core-ui";
+import {
+  ICounterfactualData,
+  IDataset,
+  JointDataset
+} from "@responsible-ai/core-ui";
 import { localization } from "@responsible-ai/localization";
 
 export function getOriginalData(
   index: number,
   jointDataset: JointDataset,
-  dataset: IDataset
+  dataset: IDataset,
+  ifEnableLargeData: boolean,
```

Comment:
We don't need the `ifEnableLargeData`. We can directly apply the IfEnableLargeData() on dataset.

Reply:
Great catch! updated